backend/app/assets/javascripts/spree/backend/addresses.js

```diff
@@ -3,7 +3,7 @@
 Spree.ready(function() {
   "use strict";
 
-  _.each(document.querySelectorAll('.js-stock-locations-form'), function(el) {
+  _.each(document.querySelectorAll('.js-locations-form'), function(el) {
```

Comment:
Could we also rename the class to `js-addresses-form`?

Reply:
@tvdeyen Good idea! Change made. Thanks!